system-variables.md

```diff
@@ -4679,6 +4679,17 @@ SHOW WARNINGS;
 - By default, Regions are split for a new table when it is being created in TiDB. After this variable is enabled, the newly split Regions are scattered immediately during the execution of the `CREATE TABLE` statement. This applies to the scenario where data need to be written in batches right after the tables are created in batches, because the newly split Regions can be scattered in TiKV beforehand and do not have to wait to be scheduled by PD. To ensure the continuous stability of writing data in batches, the `CREATE TABLE` statement returns success only after the Regions are successfully scattered. This makes the statement's execution time multiple times longer than that when you disable this variable.
 - Note that if `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` have been set when a table is created, the specified number of Regions are evenly split after the table creation.
 
+### tidb_schema_version_cache_limit <span class="version-mark">New in v7.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Default value: `16`
+- Range: `[2, 255]`
+- This variable specifies the schema caches size in the TiDB instance. The default value is '16', which means TiDB caches 16 snapshot schemas default.
+- Generally, you don't need to modify this variable. When the [Stale Read](/stale-read.md) feature is used and the DDL is executed very frequently, it will cause the schema version to change very frequently, resulting in Stale Read taking a lot of time to rebuild the snapshot schema information due to schema cache misses when getting snapshot schema. At this point, you can increase the value of `tidb_schema_version_cache_limit` (for example, 32) to avoid the problem of schema cache misses.
+- Modifying this variable causes the memory usage of TiDB to increase slightly. You should pay attention to the memory usage of TiDB to avoid OOM problems. 
```

Comment:
```suggestion
- This variable specifies the schema caches size in the TiDB instance. The default value is '16', which means TiDB caches 16 historical versions of schema information.
- Generally, you do not need to modify this variable. When the [Stale Read](/stale-read.md) feature is used and the DDL is executed very frequently, it will cause the schema version to change very frequently, resulting in Stale Read taking a lot of time to rebuild the snapshot schema information due to schema cache misses when getting snapshot schema. In this case, you can increase the value of `tidb_schema_version_cache_limit` (for example, `32`) to avoid the problem of schema cache misses.
- Modifying this variable causes the memory usage of TiDB to increase slightly. Monitor the memory usage of TiDB to avoid OOM problems. 
```